recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend

```diff
@@ -0,0 +1,13 @@
+do_install() {
+    # Install header files
```

Comment:
can we instead modify the original recipe and copy OpenCL files only if they are present?

Reply:
Done